OneCommunityGlobal/HGNRest
# Christy Unit tests for the Team Controller's getAllTeams and getTeamById

# Description
Unit tests for the getAllTeams, getTeamById, and postTeam functions in the teamController

## Related PRS (if any):
N/A

## Main changes explained:
- Unit test for the getAllTeams and getTeamById function in the teamController

## How to test:
1. check into current branch
2. do `npm install` 
3. run `npm run test teamController`
4. verify all tests pass

## Screenshots or videos of changes:

![image](https://github.com/OneCommunityGlobal/HGNRest/assets/59427068/b6898d49-b93f-4fc8-9ef7-f2e22fca51c3)
